docker-compose.yml

```diff
@@ -57,3 +59,54 @@ services:
       - "4318:4318"
     environment:
       COLLECTOR_ZIPKIN_HOST_PORT: ":9411"
+
+  langfuse:
+    image: langfuse/langfuse:2.87.0
+    restart: always
+    depends_on:
+      langfuse_db:
+        condition: service_healthy
+    ports:
+      - "3000:3000"
+    healthcheck:
+      test: ["CMD-SHELL", "curl -f http://localhost:3000/api/public/health"]
+      interval: 3s
+      timeout: 3s
+      retries: 10
+    environment:
+      - DATABASE_URL=postgresql://postgres:postgres@langfuse_db:5432/postgres
+      - NEXTAUTH_SECRET=secret # generate secret with at least 256 entropy using `openssl rand -base64 32`.
+      - SALT=salt # generate secret with at least 256 entropy using `openssl rand -base64 32`.
+      - ENCRYPTION_KEY=0000000000000000000000000000000000000000000000000000000000000000 # generate secret with at least 256 bits and 64 characters in hex format using `openssl rand -hex 32`.
+      - NEXTAUTH_URL=http://localhost:3000 # URL of the frontend
+      - LANGFUSE_INIT_ORG_ID=mindsdb # Organization ID
+      - LANGFUSE_INIT_ORG_NAME=MindsDB # Organization name
+      - LANGFUSE_INIT_PROJECT_ID=23152f1d-9604-4629-a69d-27790d47b7fa # Project ID (UUID)
+      - LANGFUSE_INIT_PROJECT_NAME=MindsDB # Project name
+      - LANGFUSE_INIT_PROJECT_PUBLIC_KEY=pk-lf-1234567890 # Project public key
+      - LANGFUSE_INIT_PROJECT_SECRET_KEY=sk-lf-1234567890 # Project secret key
+      - LANGFUSE_INIT_USER_EMAIL=admin@langfuse.com # User email
+      - LANGFUSE_INIT_USER_NAME=User # User name
+      - LANGFUSE_INIT_USER_PASSWORD=password # User password
+
+  langfuse_db:
```

Comment:
consider refactoring to a single db (also for all other DBs)

Reply:
Done! PTAL?